### ISSUE TITLE
[06-06-2019] Lighthouse results for dev server

### ISSUE THREAD

**Issue by pdehaan** [mozilla/blurts-server#1079]
**TL;DR:**
- **Performance** was usually yellow, but can be tricky to test since hitting the server again could give you better perf results (and this is a dev server).
- **Accessibility** was generally good, but a could use some tweaking on /breaches page.
- **Best Practices** looks misleading, as it's flagging our use of HTTP/1.1 vs HTTP/2 for loading resources, and an "unsafe" link to cross-origin destinations in the Mozilla footer link, and the fact that the dev server spews browser console errors hitting the FxA endpoint.
- **SEO** results were always great.

Also possibly worth noting was that I'm not sure how to automate or test Lighthouse scanning with pages behind forms, so I can't seemingly test the /scan endpoint performance.

---

### Homepage

[74 Performance | 97 Accessibility | 79 Best Practices | 100 SEO](https://googlechrome.github.io/lighthouse/viewer/?gist=a42854e086b903f01975eb6dd88751a1)

```sh
$ npx lighthouse https://fx-breach-alerts.herokuapp.com/ --emulated-form-factor=desktop --view --quiet
```

---

### /?breach=LinkedIn

[87 Performance | 96 Accessibility | 79 Best Practices | 100 SEO](https://googlechrome.github.io/lighthouse/viewer/?gist=7ad18818fb0b53b4cd52ea039f89ef31)

```sh
$ npx lighthouse https://fx-breach-alerts.herokuapp.com/\?breach\=LinkedIn --emulated-form-factor=desktop --view --quiet
```

---

### /breaches

[70 Performance | 72 Accessibility | 79 Best Practices | 100 SEO](https://googlechrome.github.io/lighthouse/viewer/?gist=e29f27fcf176805909f2b5c450747a7d)

```sh
$ npx lighthouse https://fx-breach-alerts.herokuapp.com/breaches --emulated-form-factor=desktop --view --quiet
```

---

### /security-tips

[67 Performance | 96 Accessibility | 79 Best Practices | 100 SEO](https://googlechrome.github.io/lighthouse/viewer/?gist=f542bb4e4f97b850c91f24f8e6f1e150)

```sh
$ npx lighthouse https://fx-breach-alerts.herokuapp.com/security-tips --emulated-form-factor=desktop --view --quiet
```

---

### /about

[83 Performance | 96 Accessibility | 79 Best Practices | 100 SEO](https://googlechrome.github.io/lighthouse/viewer/?gist=f58e1ec9c1c22845d440ad418fe53350)

```sh
$ npx lighthouse https://fx-breach-alerts.herokuapp.com/about --emulated-form-factor=desktop --view --quiet
```

**Comment by pdehaan** [comment]
A bit more digging into the differences between the development and production server results.

For the https://monitor.firefox.com/security-tips page, I'm getting the following results:

![Lighthouse_Report_Viewer](https://user-images.githubusercontent.com/557895/59057498-5c122e00-884f-11e9-8711-45e8db3790d2.png)
**Figure 1:** [Lighthouse Report...](https://googlechrome.github.io/lighthouse/viewer/?gist=29a4e3cf81be01c3c198de790869da78)

## Results:

### Accessibility
1. `<html>` element does not have a `[lang]` attribute.

### Best Practices
1. Links to cross-origin destinations are unsafe
	> Add `rel="noopener"` or `rel="noreferrer"` to any external links (https://www.mozilla.org) to improve performance and prevent security vulnerabilities. 
1. Browser errors were logged to the console
    > …landing/background-noodle-top3.svg (monitor.firefox.com) | Failed to load resource: the server responded with a status of 404 (). Ref #1015
